pkg/common/keys.go

```diff
@@ -21,6 +21,7 @@ var (
 	schedulerContainerLock           string = "scheduler:container:lock:%s"
 	schedulerContainerExitCode       string = "scheduler:container:exit_code:%s"
 	schedulerCheckpointState         string = "scheduler:checkpoint_state:%s:%s"
+	schedulerStubUnhealthyState      string = "scheduler:stub:unhealthy_state:%s"
```

Comment:
i feel like this should be scheduleStubState (and it could contain healthy contain count, and failed containers), that way client can infer what the state means. to me that sounds cleaner than "unhealthy" state

Reply:
that makes sense. However i was only planning on having this state if the state was non consider healthy. 
If this doesn't exist, the state should be healthy

Reply:
yeah the name just feels wrong imo